secrets/secrets.go

```diff
@@ -71,30 +74,91 @@ var NewKeeper = newKeeper
 // newKeeper creates a Keeper.
 func newKeeper(k driver.Keeper) *Keeper {
 	return &Keeper{
-		k: k,
-		tracer: &oc.Tracer{
-			Package:        pkgName,
-			Provider:       oc.ProviderName(k),
-			LatencyMeasure: latencyMeasure,
-		},
+		k:      k,
+		tracer: otel.GetTracerProvider().Tracer(pkgName),
 	}
 }
 
 const pkgName = "gocloud.dev/secrets"
 
 var (
-	latencyMeasure = oc.LatencyMeasure(pkgName)
+	// Meter is the OpenTelemetry meter for this package
+	meter = otel.GetMeterProvider().Meter(pkgName)
 
-	// OpenCensusViews are predefined views for OpenCensus metrics.
-	// The views include counts and latency distributions for API method calls.
-	// See the example at https://godoc.org/go.opencensus.io/stats/view for usage.
-	OpenCensusViews = oc.Views(pkgName, latencyMeasure)
+	// latencyHistogram measures the latency of method calls
+	latencyHistogram metric.Float64Histogram
+
+	// completedCallsCounter counts the number of method calls
+	completedCallsCounter metric.Int64Counter
 )
 
+func init() {
+	var err error
+	latencyHistogram, err = meter.Float64Histogram(
+		pkgName+"/latency",
+		metric.WithDescription("Latency distribution of method calls"),
+	)
+	if err != nil {
+		otel.Handle(err)
```

Comment:
What does this do, and in what state does it leave the latencyHistogram?

Reply:
The otle.Handle(err) is a global handler for errors so for sure there will be an error, however now I see we could endup with a partially configured  latencyHistogram.  Just looking further into this.

Reply:
What does "there will be an error" mean though? Does the application crash? Or does it just log something and return? If the latter, what happens later one when we try to record metrics?